packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderItem/ConditionBuilderItemOption/ConditionBuilderItemOption.js

```diff
@@ -54,6 +55,20 @@ export const ConditionBuilderItemOption = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
+    //this will focus the first input field in the popover
+
+    if (contentRef.current) {
+      const firstFocusableElement =
+        contentRef.current?.querySelector('input, button,li');
+
+      if (firstFocusableElement) {
+        firstFocusableElement.focus();
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [allOptions]);
+
   const handleSelectAll = (e) => {
     if (e.currentTarget.dataset.selectedAll == 'false') {
       onChange(undefined);
```

Comment:
Is this necessary to pass `undefined` explicitly?

Reply:
Yes , if value is `undefined` , _+_ is displayed , otherwise that cell wont be displayed while adding a condition.
But this will be changed as part of next PR  to improve readability.